changelogs/unreleased/7026-agent-install-dependency-modules.yml

```diff
@@ -0,0 +1,8 @@
+---
+description: The project configuration option `agent_install_dependency_modules` is now enabled by default.
```

Comment:
The wording may or may not have to change a bit depending on how my comment on the option resolves.